frontend/src/components/pages/InviteUsers.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import { Button, Input, Box, Text, Badge, Select } from "@chakra-ui/react";
+
+import NavigationBar from "../common/NavigationBar";
+import commonApiClient from "../../APIClients/CommonAPIClient";
+
```

Comment:
We can use the following const here and map through the values later when rendering the options (refer to below comment for follow-up code)

```suggestion
const ROLE_OPTIONS = ['Relief Staff', 'Admin', 'Regular Staff']
```